deobfuscator-api/src/main/java/org/objectweb/asm/tree/analysis/OriginalSourceValue.java

```diff
@@ -48,9 +50,29 @@ public class OriginalSourceValue extends SourceValue {
    */
   public final OriginalSourceValue originalSource;
 
-  public OriginalSourceValue(OriginalSourceValue copiedFrom, AbstractInsnNode insnNode) {
-    this(copiedFrom.size, Set.of(insnNode), copiedFrom);
-  }
+  /**
+   * Predicted constant value. It holds a constant value such as {@link Integer}, {@link Double},
+   * {@link Float}, {@link String}, {@link Type} and {@code null}. It also follows all math operations
+   * and jumps to get the constant value.
+   *
+   * <p>
+   * Consider this example:
+   * <pre>
+   * 1: A:
+   * 2:   ldc 12L
+   * 3:   ldiv
+   * 4:   l2i
+   * 5:   lookupswitch {
+   * 6:     ...
+   * 7:   }
+   * </pre>
+   *
+   * In line 2, the constant value is 12L. In line 3, the constant value is 12L / 2L = 6L. In line 4,
```

Comment:
example misses loading cosntant 2 for division

Reply:
Oh, you are right. I will add it to my existing PR